.github/workflows/push_pr.yml

```diff
@@ -32,6 +32,8 @@ jobs:
       run: pip install -r requirements/requirements.txt
     - name: Development requirements
       run: pip install -r requirements/requirements.dev.txt
+    - name: Install MX2
+      run: pip install --pre -f https://dist.mxnet.io/python mxnet==2.0.0b20210707 --upgrade 
```

Comment:
this works for macos builds, but for some reason the linux nightly build for that date hasn't been published. will monitor this.